tests/components/airly/test_config_flow.py

```diff
@@ -84,12 +84,13 @@ async def test_create_entry(hass, aioclient_mock):
     )
     aioclient_mock.get(API_POINT_URL, text=load_fixture("airly_valid_station.json"))
 
-    result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": SOURCE_USER}, data=CONFIG
-    )
+    with patch("homeassistant.components.airly.async_setup_entry", return_value=True):
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": SOURCE_USER}, data=CONFIG
+        )
 
-    assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
-    assert result["title"] == CONFIG[CONF_NAME]
-    assert result["data"][CONF_LATITUDE] == CONFIG[CONF_LATITUDE]
-    assert result["data"][CONF_LONGITUDE] == CONFIG[CONF_LONGITUDE]
-    assert result["data"][CONF_API_KEY] == CONFIG[CONF_API_KEY]
+        assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
```

Comment:
The assertions don't need to be inside the context manager.

Reply:
Yes, of course you're right :)